src/server/index.ts

```diff
@@ -27,3 +27,23 @@ const port = 8001;
 app.listen(port, () => {
   logInfo(`Server now listening on port: ${port}`);
 });
+
+app.use(function (req, res, next) {
+  const acceptedOrigins = [
+    'http://localhost:8033',
+    'http://localhost:3000',
+    'https://familie-ef-blankett.intern.nav.no',
+  ];
+  const defaultAcceptedOrigin = 'https://familie-ef-blankett.intern.nav.no';
+  const origin = req.header('origin')?.toLowerCase();
+
+  res.setHeader(
+    'Access-Control-Allow-Origin',
+    origin && acceptedOrigins.includes(origin) ? origin : defaultAcceptedOrigin,
+  );
+  res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
+  res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
+  res.setHeader('Access-Control-Allow-Credentials', 'true');
+
+  next();
```

Comment:
Dette gir ikke lenger mening, siden det bare er server-server for vår del... Kan slette hele bolken